WolvenKit.CLI/Commands/BuildCommand.cs

```diff
@@ -0,0 +1,37 @@
+using System.CommandLine;
+using System.CommandLine.NamingConventionBinder;
+using System.IO;
+using CP77Tools.Tasks;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using WolvenKit.Core.Interfaces;
+
+namespace CP77Tools.Commands;
+
+internal class BuildCommand : CommandBase
+{
+    private const string s_description = "Build project(s) like WolvenKit application.";
```

Comment:
Should say something like `Builds all WolvenKit .cpmodproj project(s) found in <paths>`. We should double check if `[path]` or `<path>` is better. But you get the idea.